craft_cli/messages.py

```diff
@@ -44,15 +45,25 @@ def get_terminal_width():
     return shutil.get_terminal_size().columns
 
 
+def get_log_filepath(appname):
```

Comment:
```suggestion
def get_log_filepath(appname: str):
```

Reply:
there are no tests for this method

Reply:
**this will change heavily in next couple of branches**

Tests exist in the "big PR".